webpack.config.js

```diff
@@ -1,61 +1,7 @@
-const path = require('path');
+// @flow
+const { targets, createConfig } = require('./webpack.common');
```

Comment:
Would we want to rename this to `webpack.prod.js`?

Reply:
@rrandallcainc As `webpack`'s default configuration filename is `webpack.config.js`, if `webpack` is called without any arguments, production build will be run.